src/plugins/auto/src/auto_compiled_model.cpp

```diff
@@ -48,21 +48,15 @@ ov::Any AutoCompiledModel::get_property(const std::string& name) const {
                                                     ov::device::priorities,
                                                     ov::device::properties,
                                                     ov::hint::model_priority,
-                                                    ov::loaded_from_cache};
+                                                    ov::loaded_from_cache,
+                                                    ov::enable_profiling};
         return ro_properties;
     };
     const auto& default_rw_properties = []() {
         std::vector<ov::PropertyName> rw_properties{};
         return rw_properties;
     };
-    const auto& to_string_vector = [](const std::vector<ov::PropertyName>& properties) {
-        std::vector<std::string> ret;
-        for (const auto& property : properties) {
-            ret.emplace_back(property);
-        }
-        return ret;
-    };
-    if (name == ov::supported_properties) {
+    if (name == ov::supported_properties || name == ov::enable_profiling) {
```

Comment:
should not be added here?
also please enable same for auto ctput and add some tests

Reply:
@panhaiqi  please add a new item for ov::enable_profiling.